tasks/scripts/build-release-resource-versions

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -e
+
+echo "untar-ing rc"
+tar -zxf ./linux-rc/*.tgz concourse/resource-types --strip-components=1
+
+FILE=resource-versions/versions
+
+cat > $FILE <<EOF
+
+## 📦 Bundled resource types
```

Comment:
I think in the future this should be something like `📦 Bundled assets` or `📦 Bundled dependencies`, but it makes sense for now. Maybe we don't need the `Base resource types` dropdown though while it's in the section ` 📦 Bundled resource types`, though?

Reply:
> I think in the future this should be something like 📦 Bundled assets or 📦 Bundled dependencies

I like `📦 Bundled dependencies`, if we end up wanting to keep track of go and gdn versions, this could also be something like `🔨 Build configuration`

> Maybe we don't need the Base resource types dropdown though

My thought was that this might be a small thing that only very few people would care about, in which case it would be nice for it to not take up too much space and only show up when you explicitly click on it.

Reply:
Makes sense. I think if the header is `Bundled resource types`, the default `Details` dropdown is appropriate.

Personally, I don't think we should use `Bundled dependencies` until we start documenting *all* the dependencies, so I think keeping it as `Bundled resource types` for now is best

Reply:
Done